src/mapping/attractor_mapping_recurrences.jl

```diff
@@ -313,6 +376,47 @@ function automatic_Δt_basins(ds, grid; N = 5000)
     return Δt
 end
 
+function subdivision_based_grid(ds, grid; maxlevel = 4)
```

Comment:
This function is exported but it does not have a documentation string. Not allowed.